tests/apiserver-availability-test.yaml

```diff
@@ -0,0 +1,23 @@
+rule_files:
+- ../prometheus_alerts.yaml
+- ../prometheus_rules.yaml
+
+evaluation_interval: 1m
+
+tests:
+- interval: 1m
+  input_series:
+  # 100 requests in the overall interval, 99 successful and 1 error
+  - series: 'apiserver_request_total{job="kube-apiserver",verb="GET",code="200"}'
+    values: '0 10 20 50 90 99'
+  - series: 'apiserver_request_total{job="kube-apiserver",verb="GET",code="500"}'
+    values: '0 0 0 1 1 1'
+
+  promql_expr_test:
+  - eval_time: 5m
```

Comment:
btw I just found out that you can name tests, this is really useful for identifying which test has failed and can indicate the intention of the test, in case you'd like to use the `name` property ([see docs](https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/#test_group)).

Reply:
Good finding! Will start using it